astropy/io/ascii/tests/test_read.py

```diff
@@ -1034,24 +1034,40 @@ def test_almost_but_not_quite_daophot():
     assert len(dat) == 3
 
 
-@pytest.mark.parametrize('fast', [True, False])
+@pytest.mark.parametrize('fast', [True, False, 'force'])
```

Comment:
In this case `True` will use the fast reader so `force` is just re-running the same test (and burning a little test time).  But it might be a good idea to replace `True` with `'force'` so that if the fast reader failed for some reason then it would raise an exception instead of falling back to the Python reader.

Reply:
Yes, that's what was happening in the first tests....